src/utils/types.ts

```diff
@@ -278,6 +283,18 @@ export const pluginSettingItems:PluginSettingItems = {
 		section: settingSectionName,
 	},
 
+	cardInsertLocation: {
+		label: 'Add new cards to',
```

Comment:
Rename to "New card placement" please (I think it's better not to have incomplete sentences as setting labels)